src/chrome/background.ts

```diff
@@ -73,6 +78,8 @@ chrome.contextMenus.create(clipboardMenuItem);
 chrome.contextMenus.create(decryptMenuItem);
 
 chrome.contextMenus.onClicked.addListener( async (clickData) => {
+    //let {push, goBack} = useHistory();
```

Comment:
@bindeman Running into issues trying to use any react elements in the background script. `Error: Minified React error #321` Just trying to see if I can't use the results page instead of using alert() for the right click actions

Reply:
Hmmm, not sure tbh, maybe we can troubleshoot tomorrow